tests/dogtag/pytest-ansible/pytest/performance_test/test_acme_cert_enrollment.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/env python
+
+import argparse
+import subprocess
+import json
+import os, sys
+import base64
+import binascii, time
+import hashlib
+import re, logging
+import threading
+import OpenSSL
+from acme import crypto_util
+from timeit import default_timer as timer
+from urllib.request import urlopen, Request
+
+DEFAULT_DIRECTORY_URL = "http://pki1.example.com:8080/acme/directory"
```

Comment:
To make the script more useful in general, let's use `localhost` instead of `pki1.example.com` as the default.